schedule/src/main/java/itcast/message/dto/request/MessageContent.java

```diff
@@ -0,0 +1,12 @@
+package itcast.message.dto.request;
+
+import jakarta.validation.constraints.NotBlank;
+
+public record MessageContent(
+        @NotBlank
+        String title,
+        @NotBlank
+        String summary,
+        @NotBlank
+        String originalLink
+){ }
```

Comment:
record 여기 한번 띄어쓰는게 좋을거같아요. 
```suggestion
){ 
}
```